forem/forem
# Rename feeds + update feeds' tabs UI

## What type of PR is this? (check all applicable)


## Description

This PR updates the Home feed and the tagged articles feed (AKA tag index page) to use the word "Relevance" in place of "Feed" for the default sorting. It also updated the tagged articles feed to use the new feed style we're already using on the home feed.

## Related Tickets & Documents

Closes https://github.com/forem/rfcs/issues/311

## QA Instructions, Screenshots, Recordings

- Visit the home page, and check that "Feed" is now "Relevance"
- Experiment with the sort options and check all is working as before
- Visit a tag page e.g. `/t/discuss` and check the feed sorting UI is the same as the home feed
- Experiment with the sort options and check all is working as before
- Check the mobile layout to see the `select` element is gone, and the same tabbed navigation can be used

As a small aside, we know that the hamburger menus on either side of the tagged articles mobile layout are not ideal, but there are plans to get rid of these sidebars in future and we felt we can live with this for now.

**Desktop view:**

![tagged articles layout - desktop](https://user-images.githubusercontent.com/20773163/142621920-cbc09911-5326-42fa-bd3f-8b34d51844c0.png)

**Mobile view:**

![tagged articles layout - mobile](https://user-images.githubusercontent.com/20773163/142621968-2f330b76-83f0-44b5-88df-5f64a0c34c4a.png)


### UI accessibility concerns?

This PR removes the visible `<h1>` element on the tagged articles feed. It isn't ideal for accessibility to be missing a clear heading level one, but we have at least introduced a "screen reader only" one.

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_
